### ISSUE TITLE
Clara ?

### ISSUE THREAD

**Issue by tonimueller** [LeMonADE-project/LeMonADE#94]
Do we want to ship clara with our software? (Is it possible with the license?) 
At least it would be a nice feature to have something like that.

**Comment by Bondoki** [comment]
Yes, definitely as Clara is a single header files and shipped with the Boost License: see 
[Bondoki/ELMA](https://github.com/Bondoki/ELMA/tree/WangLandau/include/extern/catchorg/clara)
Maybe two drawbacks: it's not so regularly updated and it uses the C+011 (but anyway we should use that standard asap)

**Comment by tonimueller** [comment]
Seems that clara is not mantained anymore. We could use lyra [https://github.com/bfgroup/Lyra](url)
